docs/ShowGuesser.md

```diff
@@ -30,7 +30,7 @@ It also dumps the components it has guessed in the console, where you can copy i
 
 `<ShowGuesser>` doesn't expose any prop, as it's not designed to be customized. You should replace it with a custom `<Show>` component as soon as you need to customize it.
 
-React-admin provides guessers for the `List` view (`ListGuesser`), the `Edit` view (`EditGuesser`), and the `Show` view (`ShowGuesser`).
+React-admin provides guessers for the `List` view ([`<ListGuesser>`](./List.md#the-listguesser-component)), the `Edit` view ([`<EditGuesser>`](./CreateEdit.md#the-editguesser-component)), and the `Show` view (`ShowGuesser`).
```

Comment:
Missing the `Show` and `ShowGuesser`

Reply:
Bur we are in the ShowGuesser doc page...